clash_lib/src/config/internal/proxy.rs

```diff
@@ -368,3 +371,53 @@ impl TryFrom<HashMap<String, Value>> for OutboundProxyProviderDef {
             .map_err(map_serde_error)
     }
 }
+
+//   #hysteria2
```

Comment:
would be great if you can convert these into the standard doc comment https://doc.rust-lang.org/rust-by-example/meta/doc.html

so that we might use them in future to generate the API references